templates/_partials/microdata/product-jsonld.tpl

```diff
@@ -17,6 +17,26 @@
       "@type": "Brand",
       "name": "{if $product_manufacturer->name}{$product_manufacturer->name|escape:'html':'UTF-8'}{else}{$shop.name}{/if}"
     },{/if}
+    {if !empty($product.productComments.comments)}
```

Comment:
As I mentioned in the github issue I think the idea is good but also I think we should try to expose a variable that is NOT related to productcomment module. This allows people to remove the productcomment module if they want to and to be able to inject this variable.

Something like `product.reviewData` ?

Reply:
I guess it will even throw a 500 in case the module has been deleted nop?

Reply:
We should say how the structure should look, and review modules can include it in hookProductLazyArray.

Reply:
> @rodriciru you need to check that the variable is defined too

If I uninstall, or disable the module, nothing bad is happening. Simply it ignores the comments part and the review, but it won't break